recipes/pyRingGW/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyRingGW" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyRingGW-{{ version }}.tar.gz
+  sha256: f354b2ee1104d09091c1ea1ca7f1d05ad08aa8f0fd1c09bc0a8605917a6b5ad6
+
+build:
+  skip: true  # [py<36]
+  entry_points:
+    - pyRing = pyRing.pyRing:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython
+    - numpy
+    - pip
+  run:
+    - python
+    - corner
+    - cpnest
+    - cython
```

Comment:
Why do you need cython at runtime? Typically once the cython files have been translated to c and compiled into shared objects, cython is not needed anymore.